roamer/session.py

```diff
@@ -33,5 +34,21 @@ def process(self, output):
         engine = Engine(self.directory, self.edit_directory)
         engine.compile_commands()
         print(engine.commands_to_str())
+        if engine.commands:
+            answer = 'y'
+            if not self.skipapproval:
+                print(
+                    'Argument --skip-approval could be used to run roamer '
+                    'in a noninteractive mode.'
+                )
+                try:
+                    answer = input('Please indicate approval: [y/N] ')
```

Comment:
This line is only compatible with python 3.x.  We need to use `raw_input` to support python 2.x.
```
if sys.version_info[0] == 3:
    answer = input('Please indicate approval: [y/N] ')
else:
    answer = raw_input('Please indicate approval: [y/N] ')

```

Reply:
And yeah, I've forgot to do that. Maybe top of the file is the better place for that?